pkg/web/apps/apps_psql_repository.go

```diff
@@ -19,16 +22,59 @@ func NewPostgreSQLRepository(db *sql.DB) Repository {
 
 // GetApps retrieves all apps from the database
 func (a *appsPostgreSQLRepository) GetApps() (*[]models.App, error) {
-	app1 := models.App{
-		ID:                    "a0874c82-2b7f-11e9-b210-d663bd873d93",
-		AppID:                 "com.aerogear.app1",
-		AppName:               "app1",
-		NumOfDeployedVersions: 1,
-		NumOfClients:          1,
-		NumOfAppLaunches:      1,
+	rows, err := a.db.Query(`SELECT id,app_id,app_name FROM app WHERE deleted_at IS NULL;`)
+
+	if err != nil {
+		log.Error(err)
```

Comment:
@davidffrench it is a scenario that we could raise the error msg. For example, if the error msg here be "no connection" then this error msg could be thrown to the handler and returned in the response instead of the empty string. wdyt?

Reply:
Where are you seeing the empty string? There is a current drawback of the generic models that there is no current way to input your own string.

Reply:
Wdyt over create a task for us to improve the generic model's error class and allow it? IHMO it is a high import for a dev get the error message to know what is going wrong and be able to solve it.

Reply:
@camilamacedo86 I agree this should be done at some point. What we have now is limited. [Here is](https://medium.com/@sebdah/go-best-practices-error-handling-2d15e1f0c5ee) a good implementation I came across yesterday.

Reply:
@craicoverflow good post. I will create the JIRA and add it there so. tks for the support

Reply:
https://issues.jboss.org/browse/AEROGEAR-8542